tests/core/Statiq.Common.Tests/IO/NormalizedPathFixture.cs

```diff
@@ -1456,6 +1456,23 @@ public void OptimizesInstance()
                 // Then
                 result.FullPath.ShouldBe("a/b/c/abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz0123456789.html");
             }
+
+            [TestCase("2020.09.29.1.html", "2020.09.29.1")]
+            [TestCase("Statiq.Framework.sln.DotSettings", "Statiq.Framework.Sln")]
```

Comment:
Expected should be `Statiq.Framework.sln`

Reply:
It does title case for document titles, haven't changed that behavior

Reply:
Ah, ok. Didn't know. Thought it was a typo.

Reply:
Thanks for keeping an eye out @patriksvensson!